changelog/fix-3853-deposits-evidence-upload-preview

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: tweak
+Comment: Fixing a feature that is not released yet.
```

Comment:
That's looks different from what we have in other files, can you ensure the format is correct, please?

Reply:
It is correct. The `Comment` is used for internal comment only as we don't need a public changelog entry.